simplq/src/styles/buttons.module.scss

```diff
@@ -10,16 +10,17 @@
   border: 2px solid $primary-color-dark;
   border-radius: 0.6rem;
   text-align: center;
-  transition: all 0.2s;
+  transition: background 0.8s;
+  background-position: center;
 
   &:hover {
-    background-color: #fff;
-    color: $primary-color-dark;
-    cursor: pointer;
+    background: #776fff radial-gradient(circle, transparent 1%, #776fff 1%) center/15000%;
   }
```

Comment:
The ripple effect looks good, but I don't really like the color on hover (this is a personal opinion). Maybe something a bit darker to match the primary dark color would be better. @daltonfury42 Any thoughts on the color.